splitmix.cabal

```diff
@@ -39,9 +39,10 @@ tested-with:
      || ==9.0.2
      || ==9.2.8
      || ==9.4.8
-     || ==9.6.4
-     || ==9.8.2
+     || ==9.6.6
+     || ==9.8.4
      || ==9.10.1
+     || ==9.11.1
```

Comment:
Was this supposed to be `9.12.1`?  
Haskell-CI should have complained...

Reply:
It doesn't, as it forgets the structure of tested-with (treats it as a true `VersionRange`, not as set of discrete points).I probably should change that, I remember someone else complaining about this behavior already.